spec/features/registration_spec.rb

```diff
@@ -0,0 +1,38 @@
+feature "registration" do 
+
+  scenario "create a new user" do
+    go_to_index_page_and_register_a_new_user
+    new_user = User.last
+    expect(new_user.username).to eq "tester"
+  end
+
+  scenario "starting a new session" do
+    go_to_index_page_and_register_a_new_user
+
+    expect(current_path).to eq "/"
+    click_button "Sign in"
+    expect(current_path).to eq "/sessions/new"
+
```

Comment:
I know this is using multiple web helpers but maybe could be refactored further?